veryfi/client.go

```diff
@@ -71,7 +81,7 @@ func (c *Client) ProcessDocumentUpload(opts scheme.DocumentUploadOptions) (*sche
 // ProcessDocumentUploadBase64 returns the processed base64 encoded document.
 func (c *Client) ProcessDocumentUploadBase64(opts scheme.DocumentUploadBase64Options) (*scheme.Document, error) {
```

Comment:
I think it doesn't make sense to keep this one. The feature is the same as ProcessDocumentUpload